galata/test/jupyterlab/notebook-run-vega.test.ts

```diff
@@ -41,16 +60,20 @@ test.describe.serial('Notebook Run Vega', () => {
     const nbPanel = await page.notebook.getNotebookInPanel();
 
     expect(await nbPanel.screenshot()).toMatchSnapshot(imageName);
+    expect(await nbDiskContent(page, nbPath)).toContain(PNG_MIME_TYPE);
   });
 
   test('Run notebook with Vega cell in dark theme', async ({
     page,
     tmpPath
   }) => {
-    await page.notebook.openByPath(`${tmpPath}/${fileName}`);
+    const nbPath = `${tmpPath}/${fileName}`;
+    await page.notebook.openByPath(nbPath);
     await page.notebook.activate(fileName);
     await page.theme.setDarkTheme();
 
+    expect(await nbDiskContent(page, nbPath)).not.toContain(PNG_MIME_TYPE);
```

Comment:
A failure here is surprising. As if changing to dark theme had some effect on the paths?